src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js

```diff
@@ -172,6 +172,7 @@ export default class bdiObjectMappingModal extends LightningElement {
             this.getDataImportFieldDescribes();
             this.getDataImportMappedFields();
         } catch(error) {
+            console.log('Error encountered in objectMappingModal init: ' + error);
```

Comment:
For outside observers: these logs are temporary and will be removed, we just need to be able to track down the nature of the issue if our first fix doesn't tackle it. These logs will minimize future iterative merges to master.